alf/trainers/policy_trainer.py

```diff
@@ -292,22 +386,8 @@ def _train(self):
                 # We need to wait for one iteration to get the operative args
                 # Right just give a fixed gin file name to store operative args
                 common.write_gin_configs(self._root_dir, "configured.gin")
-
                 with alf.summary.record_if(lambda: True):
-
-                    def _markdownify(paragraph):
-                        return "    ".join(
-                            (os.linesep + paragraph).splitlines(keepends=True))
-
-                    common.summarize_gin_config()
-                    alf.summary.text('commandline', ' '.join(sys.argv))
-                    alf.summary.text(
-                        'optimizers',
-                        _markdownify(self._algorithm.get_optimizer_info()))
-                    alf.summary.text('revision', git_utils.get_revision())
-                    alf.summary.text('diff',
-                                     _markdownify(git_utils.get_diff()))
-                    alf.summary.text('seed', str(self._random_seed))
+                    self._summarize_training_setting()
```

Comment:
line 386-390 is same as line 529-533. And they can be put into self._summarize_training_setting()